tasks/main.yml

```
@@ -1,2 +1,31 @@
 ---
-# tasks file for skeleton
+# tasks file for apt-over-https
+
+- name: Perform any setup before altering apt repository URLs
+  ansible.builtin.include_tasks: setup.yml
+
+- name: Load the var file with regexp definition based on the OS type
+  ansible.builtin.include_tasks: "{{ lookup('first_found', params) }}"
```

You changed `_tasks` to `_var` in the comment, but not where it really counts.  I'm laughing because I do this sort of thing all the time.
```suggestion
  ansible.builtin.include_vars: "{{ lookup('first_found', params) }}"
```

---

Yep I just force-pushed to fix that oversight. *facepalm*